Add Sleepy Craft Studios Sleepy Keeb Split (#23844)
qmk/qmk_firmware: keyboards/sleepy_craft_studios/sleepy_keeb_split/readme.md

# sleepy_keeb_split

![sleepy_keeb_split](imgur.com image replace me!)

> 

Reviewer: Please fill this out.

The Sleepy Keeb Split is an ortholinear 4x6 split keyboard inspired by the Planck. This is a hand-wired keyboard using the pro-micro or similar microcontrollers. The keyboard is designed to be used with a 3d printed case, which is also available on the Sleepy Craft Studios website. The unique element to this design is the integrated hotswap holders present in the 3d printed plate. The plate also includes diode holders for easy soldering.

This keyboard is available as a kit as on the sleepycraftstudios.com, and is designed to be a beginner friendly hand-wired keyboard. The keyboard is also compatible with VIA, and the keymap is designed to be easily modified.

The STL files are made available under the CC BY-NC-SA 4.0 license.

* Keyboard Maintainer: [Sleepy Craft Studios](https://github.com/sleepy-craft-studios)
* Hardware Supported: Pro-micro (tested), Elite-C, any other ATMega32U4 based microcontroller should work.
* Hardware Availability: Kits can be purchased at [Sleepy Craft Studios](https://sleepycraftstudios.com/shop/sleepy-keeb-split-(diy))

> Make example for this keyboard (after setting up your build environment. The via keymap is recommended for VIA functionality):
> 
>     make sleepy_craft_studios/sleepy_keeb_split:via
> 
> Flashing example for this keyboard:
> 
>     make sleepy_craft_studios/sleepy_keeb_split:via:flash

Reviewer 2: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make sleepy_craft_studios/sleepy_keeb_split:default

Flashing example for this keyboard:

    make sleepy_craft_studios/sleepy_keeb_split:default:flash
```
